### PR TITLE
feat(releasesv2): banner

### PR DESCRIPTION
**Pull request by doralchan**
Screen Shot 2020-04-09 at 6 51 20 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/views/releasesV2/list/introBanner.tsx [hunk 1]
```diff
@@ -28,22 +31,27 @@ class IntroBanner extends React.Component<{}, State> {
...
-      </Banner>
+      <StyledBanner
+        title={t('Spot Release Changes')}
+        subtitle={t(
+          'See differences between releases, from crash analytics to adoption rates.'
+        )}
+        backgroundImg={backgroundLighthouse}
+        onCloseClick={this.handleBannerCloseClick}
+      >
+        <StarterButton>View Features</StarterButton>
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/releasesV2/list/introBanner.tsx:42]:
```suggestion
        <StarterButton>{t('View Features')}</StarterButton>
```

File: src/sentry/static/sentry/app/views/releasesV2/list/introBanner.tsx [hunk 2]
```diff
@@ -28,22 +31,27 @@ class IntroBanner extends React.Component<{}, State> {
...
+        <StarterButton priority="primary">Update SDK</StarterButton>
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/releasesV2/list/introBanner.tsx:43]:
```suggestion
        <StarterButton priority="primary">{t('Update SDK')}</StarterButton>
```

**Review by matejminar — APPROVED** [review]
Thanks, I will find out what those links should be and then uncomment code to call this banner. We can merge this in the meantime - it looks really cool. 🚀

**Comment by doralchan** [comment]
@matejminar Actually I dunno why the StyledBanner isn't taking in the new color

Screen Shot 2020-04-09 at 3 13 52 PM

It's supposed to look like this:
01_release_banner

**Comment by matejminar** [comment]
The target element needs to accept and pass down className prop in order for styled-components to work - I've updated the PR.

File: src/sentry/static/sentry/app/views/releasesV2/list/index.tsx [hunk 3]
```diff
@@ -220,7 +220,7 @@ class ReleasesList extends AsyncView<Props, State> {
               </SortAndFilterWrapper>
             </StyledPageHeader>
 
-            {/* <IntroBanner /> */}
+            <IntroBanner />
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/releasesV2/list/index.tsx:223]:
```suggestion
            {/* <IntroBanner /> */}
```

File: src/sentry/static/sentry/app/views/releasesV2/list/index.tsx [hunk 4]
```diff
@@ -17,7 +17,7 @@ import PageHeading from 'app/components/pageHeading';
 import withOrganization from 'app/utils/withOrganization';
 import LoadingIndicator from 'app/components/loadingIndicator';
 import LightWeightNoProjectMessage from 'app/components/lightWeightNoProjectMessage';
-// import IntroBanner from 'app/views/releasesV2/list/introBanner';
+import IntroBanner from 'app/views/releasesV2/list/introBanner';
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/releasesV2/list/index.tsx:20]:
```suggestion
// import IntroBanner from 'app/views/releasesV2/list/introBanner';
```

**Review by matejminar — COMMENTED** [review]
Let's keep this commented out please until I link those buttons because otherwise some opted in customers would actually be able to see the banner already.

File: src/sentry/static/sentry/app/views/releasesV2/list/introBanner.tsx [hunk 5]
```diff
@@ -2,9 +2,12 @@ import React from 'react';
 import styled from '@emotion/styled';
 
 import {t} from 'app/locale';
-import theme from 'app/utils/theme';
 import Banner from 'app/components/banner';
+import Button from 'app/components/button';
 import localStorage from 'app/utils/localStorage';
+import space from 'app/styles/space';
+
+import backgroundLighthouse from '../../../../images/spot/background-lighthouse.svg';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/releasesV2/list/introBanner.tsx:10]: @doralchan should I add a webpack alias for the images directory? any good ideas for an alias? `sentry-images`?

**Comment by billyvg** [comment]
This banner looks great!

File: src/sentry/static/sentry/app/views/releasesV2/list/introBanner.tsx (hunk 5 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/views/releasesV2/list/introBanner.tsx:10]: That would be great!

**Pull request merged**